internal/server/v1/firehose/mappings.go

```diff
@@ -39,10 +39,12 @@ const (
 	labelDescription = "description"
 )
 
+const logSinkTTL = 24 * time.Hour
+
 var nonAlphaNumPattern = regexp.MustCompile("[^a-zA-Z0-9]+")
 
-func mapFirehoseEntropyResource(def models.Firehose, prj *shieldv1beta1.Project) (*entropyv1beta1.Resource, error) {
-	cfgStruct, err := makeConfigStruct(def.Configs)
+func mapFirehoseEntropyResource(def models.Firehose, prj *shieldv1beta1.Project, createdAt time.Time) (*entropyv1beta1.Resource, error) {
```

Comment:
we don't need to introduce createdAt here, we can just update `models.Firehose.Configs.StopDate` before passing it to `mapFirehoseEntropyResource`  right?